src/plugins/console/server/lib/spec_definitions/json/generated/reindex_rethrottle.json

```diff
@@ -9,6 +9,6 @@
     "patterns": [
       "_reindex/{task_id}/_rethrottle"
     ],
-    "documentation": "https://www.opensearch.org/guide/en/elasticsearch/reference/master/docs-reindex.html"
+    "documentation": "https://opensearch.org/docs/latest/opensearch/rest-api/document-apis/reindex/"
```

Comment:
I don't see the rethrottle details in the documentation. I think this should be noted in the `noDocumentation` list.